test/integration/experimental-warning/test/index.test.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-env jest */
+
+import { join } from 'path'
+import { nextBuild, File } from 'next-test-utils'
+
+const appDir = join(__dirname, '../')
+const nextConfig = new File(join(appDir, 'next.config.js'))
+
+describe('experimental-warning', () => {
+  afterEach(() => {
+    nextConfig.restore()
+  })
+
+  it('should not print experimental warning if no experimental key', async () => {
+    const { code, stdout, stderr } = await nextBuild(appDir)
```

Comment:
```suggestion
    const { code, stdout, stderr } = await nextBuild(appDir, undefined, { stderr: true, stdout: true })
```

Reply:
I realized there was already an existing test so I updated that in [f170c58](https://github.com/vercel/next.js/pull/37752/commits/f170c58aabaf6fb76fe5fb799b62fef24e542d50)